policies/master_payer.json

```diff
@@ -0,0 +1,153 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "s3:Get*",
+                "s3:List*"
+            ],
+            "Resource": [
+                "arn:aws:s3:::${BillingBucketName}",
+                "arn:aws:s3:::${BillingBucketName}/*"
+            ],
+            "Effect": "Allow",
+            "Sid": "AccessMasterPayerBillingBucket"
+        },
+        {
```

Comment:
Merge this section into the `CZBudgetMonitoring20190912` section and rename `CZCostMonitoring20191111`

Reply:
also make sure this change is made to the resource owner policy as well